src/pages/SelectBallotPage.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react'
+import { RouteComponentProps } from 'react-router-dom'
+import ReactSVG from 'react-svg'
+import styled from 'styled-components'
+
+import Main, { MainChild } from '../components/Main'
+import Prose from '../components/Prose'
+
+const Legend = styled.fieldset`
+  margin: auto;
+`
+const CodeBox = styled.div`
+  max-width: 320px;
+  margin: auto;
+`
+
+let resetActivationCode: number
+
+const StartPage = (props: RouteComponentProps) => {
+  const [activationCode, setActivationCode] = useState('')
+  const goToStartPage = () => {
+    props.history.push('/start')
+  }
+  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setActivationCode(event.target.value)
+    if (event.target.value === 'MyVoiceIsMyPassword') {
```

Comment:
the scanner sends a newline by default, let's make sure we either ignore the newline or make it part of the scan value.

Reply:
Bette yet, I'll make it a proper`<form>` such that the new line causes form to submit.

Reply:
fixed.